Library/Homebrew/utils.rb

```diff
@@ -376,6 +376,14 @@ def ignore_interrupts(opt = nil)
   trap("INT", std_trap)
 end
 
+def capture_stderr
```

Comment:
Think you can use `shutup` here. Again, won't let it block this PR but for post-merge refactoring that'd be good if it's possible.